src/index.d.ts

```diff
@@ -0,0 +1,15 @@
+declare module 'htl' {
+    /** Renders the specified markup as an element, text node, or null as appropriate. */
+    const html: {
+        <T extends HTMLElement | Text | null>(...args: any[]): T;
+        /** Renders the specified markup as a document fragment. */
+        fragment(...args: any[]): DocumentFragment;
+    };
+    /** Renders the specified markup as an SVG element, text node, or null as appropriate. */
+    const svg: {
+        <T extends SVGElement | Text | null>(...args: any[]): T;
+        /** Renders the specified markup as a document fragment. */
+        fragment(...args: any[]): DocumentFragment;
+    };
```

Comment:
Slightly improved typings to have the proper tagged template string params (as per [typings for `String.raw`](https://github.com/microsoft/TypeScript/blob/main/lib/lib.es2015.core.d.ts#L558)):

```suggestion
    type TaggedTemplateParams = [
        template: { raw: readonly string[] | ArrayLike<string> },
        ...substitutions: any[],
    ];

    /** Renders the specified markup as an HTML element, text node, or null as appropriate. */
    const html: {
        <T extends HTMLElement | Text | null>(...args: TaggedTemplateParams): T;
        /** Renders the specified markup as a document fragment. */
        fragment(...args: TaggedTemplateParams): DocumentFragment;
    };
    /** Renders the specified markup as an SVG element, text node, or null as appropriate. */
    const svg: {
        <T extends SVGElement | Text | null>(...args: TaggedTemplateParams): T;
        /** Renders the specified markup as a document fragment. */
        fragment(...args: TaggedTemplateParams): DocumentFragment;
    };
```